[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/core/how-to-capture-a-memory-dump-of-a-biztalk-process.md

## Installation of the IIS Diagnostics Toolkit

>  Each of the topics in this section requires the use of the **Debug Diagnostics Tool** of the IIS Diagnostics Toolkit to capture a memory dump. To install the **Debug Diagnostics Tool** of the IIS Diagnostics Toolkit follow these steps:

Reviewer: ```suggestion
For each topic in this section, to capture a memory dump, you have to use the **Debug Diagnostics Tool** in the IIS Diagnostics Toolkit. To install this tool, follow these steps:
```

> 1.  Download the Internet Information Services (IIS) Diagnostics Toolkit (**iisdiag.msi**).

Reviewer: ```suggestion
1. Download the Internet Information Services (IIS) Diagnostics Toolkit (**iisdiag.msi**).
```

> 1.  Download the IIS Diagnostics Toolkit from Internet Information Services Diagnostic Tools.

Reviewer: ```suggestion
1.  Download the Internet Information Services (IIS) Diagnostics Toolkit.
```

> 2.  Double-click the **iisdiag.msi** package to start the IIS Diagnostics Toolkit setup program and choose the **Custom** setup type.

Reviewer: ```suggestion
2. Double-select the **iisdiag.msi** package to start the IIS Diagnostics Toolkit setup program. Choose the **Custom** setup type.
```
